internal/backuprestore/backup.go

```diff
@@ -295,17 +121,10 @@ func extractBackupFromConfigmaps(ctx context.Context, c client.Client, configmap
 				return nil, err
 			}
 
-			if resource.GroupVersionKind() != backupGvk {
+			if resource.GroupVersionKind() != BackupGvk {
 				continue
 			}
 
-			// Create the backup CR in dry-run mode to detect any validation errors in the CR
-			// i.e., missing required fields
-			err = c.Create(ctx, &resource, &client.CreateOptions{DryRun: []string{metav1.DryRunAll}})
-			if err != nil {
-				return nil, err
-			}
-
```

Comment:
Forget to mention. You need to keep this dryrun here for the "unstructured" resource type. With the CR marshalled into velerov1.backup type, the missing required fields will not be catched as they will be filled with empty string.

Reply:
Reverted